examples/advanced_operations/add_responsive_search_ad_full.py

```diff
@@ -551,10 +681,33 @@ def add_images(client, customer_id, campaign_resource_name):
         help="Whether or not the campaign will use image extensions.",
     )
 
+    # The name of the customizer attribute to be used in the ad customizer, which
+    # must be unique for a given customer account. To run this example multiple
+    # times, specify a unique value as a command line argument. Note that there is
+    # a limit for the number of enabled customizer attributes in one account, so
```

Comment:
Avoid vague admonitions. It is sufficient to state that there is a limit and use the URL.

Reply:
done